account-kit/core/src/store/store.ts

```diff
@@ -196,14 +203,20 @@ export const createSigner = (params: ClientStoreConfig) => {
     sessionConfig,
   });
 
-  const search = new URLSearchParams(window.location.search);
-  if (search.has("bundle")) {
-    signer.authenticate({ type: "email", bundle: search.get("bundle")! });
-  } else if (search.has("alchemy-bundle")) {
+  const { emailBundle, oauthBundle, oauthOrgId } = takeQueryParams({
+    emailBundle: "bundle",
+    // We don't need this, but we still want to remove it from the URL.
+    emailOrgId: "orgId",
```

Comment:
@moldy530 Is this right? I didn't see anywhere we're using the "orgId" query param.